client/core/trade.go

```diff
@@ -958,11 +810,6 @@ func (t *trackedTrade) isActive() bool {
 
 	// Status of all matches for the order.
 	for _, match := range t.matches {
-		proof := &match.MetaData.Proof
-		t.dc.log.Tracef("Checking match %s (%v) in status %v. "+
-			"Order: %v, Refund coin: %v, ContractData: %x, Revoked: %v", match,
-			match.Side, match.Status, t.ID(),
-			proof.RefundCoin, proof.ContractData, proof.IsRevoked())
```

Comment:
Change to extract from this PR.  (This TRC log was handy once when we were debugging order status issues, but now it is just noise)